### PR TITLE
fix:[PLG-441] updating plugins v1 config with wiz

### PR DESCRIPTION
**Pull request by akashpaladin**
## Description

[PLG-441]

Please include a summary of the changes and the related issues. Please also include relevant motivation and context. List
any dependencies that are required for this change.

## Type of change

**Please delete options that are not relevant.**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Chore (no code changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also
list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] My commit message/PR follows the contribution guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

## List any documentation updates that are needed for the Wiki

[PLG-441]: https://paladincloud.atlassian.net/browse/PLG-441?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by arunpaladin — APPROVED** [review]

File: installer/resources/pacbot_app/files/DB.sql [hunk 1]
```diff
@@ -3122,7 +3122,7 @@ INSERT IGNORE INTO pac_v2_ui_options (filterId,optionName,optionValue,optionURL)
 
 /* Plugin developed using PluginEngine V1 */
 INSERT INTO pac_config_properties (`cfkey`,`value`,`application`,`profile`,`label`,`createdBy`,`createdDate`,`modifiedBy`,`modifiedDate`)
- VALUES ('plugins.in.v1','gcp,redhat,contrast,checkmarx,crowdstrike','job-scheduler','prd','latest','System',now(),NULL,NULL) ON DUPLICATE KEY UPDATE value = 'gcp,redhat,contrast,checkmarx,crowdstrike', modifiedBy='System', modifiedDate=now();
+ VALUES ('plugins.in.v1','gcp,redhat,contrast,checkmarx,crowdstrike,wiz','job-scheduler','prd','latest','System',now(),NULL,NULL) ON DUPLICATE KEY UPDATE value = 'gcp,redhat,contrast,checkmarx,crowdstrike,wiz', modifiedBy='System', modifiedDate=now();
```
Comment (plyubich) [inline installer/resources/pacbot_app/files/DB.sql:3125]: Where is rapid 7?

**Review by plyubich — DISMISSED** [review]

File: installer/resources/pacbot_app/files/DB.sql (hunk 1 above)
Comment (santhosh-challa) [inline installer/resources/pacbot_app/files/DB.sql:3125]: Also should we move this as update to ee?

Comment (akashpaladin) [reply installer/resources/pacbot_app/files/DB.sql]: added rapid7

Comment (arunpaladin) [reply installer/resources/pacbot_app/files/DB.sql]: You are right Santhoh. we need to move to EE.

Comment (akashpaladin) [reply installer/resources/pacbot_app/files/DB.sql]: @arunpaladin @santhosh-challa @plyubich 
Shall I keep the value as GCP in the CE repository and move the entire query to the EE repository

Comment (santhosh-challa) [reply installer/resources/pacbot_app/files/DB.sql]: In OSS GCP is still following legacy. I think we may not need it. correct me @arunpaladin

Comment (akashpaladin) [reply installer/resources/pacbot_app/files/DB.sql]: @plyubich Rapid7 is already added, only wiz is missing

**Review by santhosh-challa — APPROVED** [review]

**Review by arunpaladin — APPROVED** [review]

**Pull request merged**